### ISSUE TITLE
Unexpected token 'A', "An error o"... is not valid JSON

### ISSUE THREAD

**Issue by zach004** [sanzodown/poe-item-checker#7]
Error message: _Unexpected token 'A', "An error o"... is not valid JSON_

Not working, attempting to search for the following item:

```
Item Class: Body Armours
Rarity: Rare
Brood Hide
Heavy Plate
--------
Armour: 632 (augmented)
--------
Requires: Level 59, 122 (unmet) Str
--------
Item Level: 80
--------
86% increased Armour
+174 to maximum Life
+18% to Fire Resistance
26.5 Life Regeneration per second
```
